marklogic-data-hub-central/src/test/resources/test-projects/reference-project/entities/Customer.entity.json

```diff
@@ -9,9 +9,11 @@
       "required": [
         "name"
       ],
+      "rangeIndex": ["customerId"],
```

Comment:
You shouldn't need this - the fact that sortable=true means a path range index will be added automatically.